test/spec_server.rb

```diff
@@ -1,3 +1,4 @@
+require 'minitest/bacon'
```

Comment:
There seems to be a need for this to be required in a `spec_helper` of sorts, thoughts?